torchvision/prototype/transforms/_type_conversion.py

```diff
@@ -33,3 +37,31 @@ def extra_repr(self) -> str:
             return ""
 
         return f"num_categories={self.num_categories}"
+
+
+class ToTensor(Transform):
+    def _transform(self, input: Any, params: Dict[str, Any]) -> Any:
+        if isinstance(input, (PIL.Image.Image, np.ndarray)):
+            return F.to_tensor(input)
+        else:
+            return input
+
+
+class PILToTensor(Transform):
+    def _transform(self, input: Any, params: Dict[str, Any]) -> Any:
+        if isinstance(input, PIL.Image.Image):
+            return F.pil_to_tensor(input)
```

Comment:
Should we convert into a `features.Image` here to "promote" the new API?

```suggestion
            output = F.pil_to_tensor(input)
            return features.Image(output, color_space=features.ColorSpace.from_pil_mode(input.mode))
```

For the user nothing changes, since a `features.Image` behaves just like a regular tensor.